contentful_management/webhooks_health_proxy.py

```diff
@@ -0,0 +1,42 @@
+from .client_proxy import ClientProxy
```

Comment:
This one is meant to be used from within the client, the other one from within the webhook itself